File: Python/Hardware.py
from Product import Product

class Hardware(Product):

    # Hardware attributes.
    __brand = ""
    __model = ""

    # Constructor.
    # Please take note that after a few try, Python seems to not accept
    # multiple constructor. So I decided to set it as empty constructor. #

    # Empty constructor.
    def __init__(self):
        pass
    
    # Setter and Getter. #

    # Set brand attribute.
    def setBrand(self, brand):
        self.__brand = brand
    
    # Set model attribute.
    def setModel(self, model):
        self.__model = model
    
    # Get brand attribute.
    def getBrand(self):
        return self.__brand
    
    # Get model attribute.
    def getModel(self):
        return self.__model
    

File: PHP/Hardware.php
<?php
    // Class Hardware, treated as midclass.
    class Hardware extends Product
    {
        // Hardware attributes.
        private $brand;
        private $model;
    
        /* Constructor.
           Please take note that after a few try, PHP seems to not accept
           multiple constructor. So I decided to set it as empty constructor. */
    
        // Empty constructor.
        public function __construct()
        {
    
        }
    
        /* Setter and Getter. */
    
        // Set brand attribute.
        public function setBrand($brand)
        {
            $this->brand = $brand;
        }
    
        // Set model attribute.
        public function setModel($model)
        {
            $this->model = $model;
        }
    
        // Get brand attribute.
        public function getBrand()
        {
            return $this->brand;
        }
    
        // Get model attribute.
        public function getModel()
        {
            return $this->model;
        }

        /* Destructor. */
        
        public function __destruct()
        {
            
        }
    }
?>